crud/select/filters.lua

```diff
@@ -127,15 +97,22 @@ local function parse(space, conditions, opts)
         if i ~= opts.scan_condition_num then
             -- Index check (including one and multicolumn)
             local fieldnos
+            local field_types
+            local values_opts
```

Comment:
Field types, but valueS opts. Probably, we need to choose one approach.

Reply:
It seems to be ok. Because value can has several options but only one type.

Reply:
No, it's not about opt and opts, it's about valueS and field. I mean we need to use `values_opts`+ `fields_types` or `value_opts` + `field_types`.

Reply:
Well. Renamed `field_types` -> `fields_types` (https://github.com/tarantool/crud/compare/078e1962a1bf4e27ccd950cd36c0fbca1a31249c..7caeffad3cc937fa7380356b553004545d715112)